tests/Mailer/Ownership/SendMailsForOwnershipEventHandlerTest.php

```diff
@@ -60,13 +56,38 @@ public function it_handles_ownership_requested_event(): void
 
         $this->commandBus->expects($this->once())
             ->method('dispatch')
-            ->with(new SendOwnershipRequestedMail($id));
+            ->with($command);
 
         $this->sendMailsForOwnership->handle(
             $domainMessage
         );
     }
 
+    public function eventToCommandProvider(): array
```

Comment:
What does "event to command" mean?

Reply:
It is the data provider for a test that checks if an Event (OwnershipApproved) is converted to a command "SendOwnershipApprovedEmail".
I renamed it to a generic "dataprovider" if it is confusing.

Reply:
Maybe `eventWithCorrespondingCommandProvider`